Component/SolidMechanics/FEM/Elastic/src/sofa/component/solidmechanics/fem/elastic/HexahedronFEMForceField.inl

```diff
@@ -747,7 +746,7 @@ void HexahedronFEMForceField<DataTypes>::initSmall(int i, const Element &elem)
         _rotatedInitialElements[i][w] =  _rotations[i]*_initialPoints.getValue()[elem[w]];
 
     auto& stiffnesses = *sofa::helper::getWriteOnlyAccessor(_elementStiffnesses);
-    if( stiffnesses.size() <= i )
+    if( stiffnesses.size() <= (Size)i )
```

Comment:
That's not how I would fix this warning.
I think `i` should be a `sofa::Size` in the first place. It implies changing the signature of the functions, and also the type of `i` to `sofa::Size` in the functions calling the `initSmall`, `initLarge` and `initPolar` functions.